server/src/api.js

```diff
@@ -194,15 +194,20 @@ app.post(
   }),
 );
 
-app.post(
```

Comment:
this was a duplicate of

https://github.com/fraktio/teamdaily/pull/62/files#diff-75bb98844a56eb2824fd5f2b8fbb9c84L184